test/unit/math/rev/functor/adj_arg_test.cpp

```diff
@@ -42,22 +42,22 @@ void test_adj_op_sizes_v(size_t n) {
     EXPECT_EQ(x_v.cols(), 1) << "Failed for type:" << type_name<T>();
     EXPECT_EQ(x_v.size(), n) << "Failed for type:" << type_name<T>();
   }
-  test_adj_op_assign_accessors_v(x_v);
+  test_adj_arg_assign_accessors_v(x_v);
```

Comment:
[optional] I like unnesting these calls because with it like this `test_adj_arg_sizes_v` tests sizes and assignment.

Seems like `test_adj_arg_sizes_v` would just test assignment.

[not optional] Also we're still missing tests of manually doing `adj_arg<T, True>`/`adj_arg<T, False>`.

Something as simple as this is all I'm looking for:

```
adj_arg<T, True> arg(std::vector<double>({ 1.0, 2.0, 3.0 });
EXPECT_NO_THROW(arg(0));
adj_arg<T, False> arg(std::vector<double>({ 1.0, 2.0, 3.0 });
EXPECT_THROW(arg(0), std::domain_error);
```

Reply:
Ah forgot about those tests! My b

I'd rather not have to duplicate code here, what if I just change the name to `test_adj_arg_sizes_and_assignment_v`?